src/graphql/queries.js

```diff
@@ -190,25 +218,39 @@ subscription {
  * Query used to retrieve data for the GScan sidebar.
  * @type {string}
  */
-const GSCAN_QUERY = `
-subscription {
-  workflows {
-    id
-    name
-    status
-    owner
-    host
-    port
-    stateTotals
-    latestStateTasks(states: [
-      "failed",
-      "preparing",
-      "submit-failed",
-      "submitted",
-      "running"
-    ])
+const GSCAN_DELTAS_SUBSCRIPTION = gql`
+subscription OnGscanDeltasData {
+  deltas (stripNull: true) {
+    ...GScanTreeDeltas
   }
 }
+
+# GSCAN DELTAS BEGIN
+
+fragment GScanTreeDeltas on Deltas {
+  id
+  shutdown
+  added {
+    ...GscanAddedData
+  }
+  updated {
+    ...GscanUpdatedData
+  }
```

Comment:
Note to self: check if there's a pruned for gscan too? Can't recall if there was one and I ignored, or it simply was never there.

Reply:
Done.

The reason why I didn't include the `pruned` was that it didn't have one for workflows. Only families, jobs, tasks, and edges.

![image](https://user-images.githubusercontent.com/304786/113039569-4b853180-91f4-11eb-9131-41fe80826f78.png)